src/components/index.js

```diff
@@ -22,27 +29,27 @@ export default function SheetJSApp(props) {
   const [form] = Form.useForm();
 
   String.prototype.capitalize = function () {
-    return this.toLowerCase().replace(/\b[a-z]/g, char => char.toUpperCase());
+    return this.toLowerCase().replace(/\b[a-z]/g, (char) => char.toUpperCase());
   };
 
   const cleanAddr = (datas, field) => {
     const addr = datas[0].indexOf(field);
     if (addr >= 0) {
       const cleaned = datas.map((item, i) => {
-        if (i == 0) return item;
+        if (i === 0) return item;
 
         item.splice(
           addr,
           1,
           item[addr]
             ? item[addr]
-                .capitalize()
-                .trim()
-                .replace(
-                  /UNKNOWN|Unknown|County|Sub County|Invalid code|[.,]+/g,
-                  ''
-                )
-            : ''
+              .capitalize()
+              .trim()
+              .replace(
+                /UNKNOWN|Unknown|County|Sub County|Invalid code|[.,]+/g,
```

Comment:
Please capitalize `'Invalid Code'` and include `'Invaild Category'` then call `.trim()` at the end to clear any resultant white spaces, as discussed earlier today.